packages/react/src/components/Tabs/Tabs.tsx

```diff
@@ -130,29 +118,25 @@ const Tabs = ({
     >
       <ul
         role="tablist"
-        className={classNames('Tablist', {
-          'Tablist--full-width': variant === 'full-width'
-        })}
+        className="Tablist"
         {...labelProp}
         onKeyDown={handleKeyDown}
       >
         {tabComponents}
       </ul>
-      {tabPanelComponents}
     </div>
   );
 };
 
 Tabs.displayName = 'Tabs';
 Tabs.propTypes = {
   children: PropTypes.node.isRequired,
-  label: PropTypes.string,
-  labelledby: PropTypes.string,
+  'aria-label': PropTypes.string,
+  'aria-labelledby': PropTypes.string,
   id: PropTypes.string,
```

Comment:
Looks like you removed the typescript property, so should probably remove this one too
```suggestion
```